manifests/i/IObit/DriverBooster/10.3.0.124/IObit.DriverBooster.yaml

```
@@ -1,9 +1,8 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
-
-PackageIdentifier: IObit.DriverBooster
-PackageVersion: 10.2.0
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.2.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster
+PackageVersion: 10.3.0.124
```

```suggestion
PackageVersion: 10.3.0
```
This should match control panel, or it will cause an upgrade loop. The version folder will need to be renamed